templates/webapi/test/infopanel.html.ep

```diff
@@ -39,28 +39,28 @@
                             % }
                         </div>
 
-                        Result: <b><%= $job->result %></b>
+                        Result: <b><%= $job->result %></b>\
                     % }
                     % else {
-                        State: <b><%= $job->state %></b>
+                        State: <b><%= $job->state %></b>\
                         % if (scalar($job->gru_dependencies) > 0) {
                             % my $grutext = '';
                             % for my $grudep ($job->gru_dependencies) {
                                 % $grutext .= join(",", $grudep->gru_task->id, $grudep->gru_task->taskname);
                                 % $grutext .= " ";
                             % }
-                            waiting for background (GRU) tasks: <%= $grutext %>
+                            , waiting for background (GRU) tasks: <%= $grutext %>
                         % }
                     % }
                     % if ($job->t_finished) {
-                        finished
+                        <%= ', finished' =%>
```

Comment:
Is it required to render the fix string like that to suppress rendering the white-spaces before? If yes, than that's missing in ", waiting for background …" and otherwise I would avoid it. Btw, I wouldn't consider it bad to simply omit the white-spaces here. Correct indentation isn't everything if it leads to surplus white-spaces being rendered.

Reply:
Yes, it's required. Yes, starting the string from the beginning of line can work as well. I consider this way nicer here as the indentation is consistent and implicitly it was "template code" in before anyway. Now it's more visible that it's code :)

Reply:
Ok, then keep it this way but apply the approach also to ", waiting for background …" .

Reply:
done